eth/handler.go

```diff
@@ -84,10 +83,16 @@ type txPool interface {
 	// Add should add the given transactions to the pool.
 	Add(txs []*txpool.Transaction, local bool, sync bool) []error
 
+	// AddBundle should add the given bundle to the pool.
+	AddBundle(bundle *types.Bundle) error
+
```

Comment:
sorry, the two interface here seems useless

Reply:
```
func (b *EthAPIBackend) SendBundle(ctx context.Context, bundle *types.Bundle) error {
	return b.eth.txPool.AddBundle(bundle)
}
```

Reply:
seems useless, the caller does not from interface, I'll remove it